test/test_plugin_helper.rb

```diff
@@ -5,14 +5,14 @@
 FactoryBot.definition_file_paths << File.join(File.dirname(__FILE__), 'factories')
 # Add factories from foreman_ansible
 FactoryBot.definition_file_paths << File.join(ForemanAnsible::Engine.root, '/test/factories')
-FactoryBot.definition_file_paths << File.join(ForemanPuppet::Engine.root, '/test/factories') if defined?(ForemanPuppet)
+FactoryBot.definition_file_paths << File.join(ForemanPuppet::Engine.root, '/test/factories') if defined?(ForemanPuppet::Engine)
```

Comment:
Just out of curiosity, Why are these changes needed if you stayed with the ```defined?``` method?

Reply:
As @ezr-ondrej mentioned: https://github.com/theforeman/foreman_openscap/pull/526#discussion_r798525310

Reply:
Than LGTM :)